Bottom: ff5c08639b97f4ef6ce60fa69464dbadfcb77e19
Top:    d292422b46f1ed26f7aac2ff246db731b7012847
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-10 20:23:24 +0000

Refresh of adjust-board-thickness-to-3

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index d38cdb3..90f281d 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -73,7 +73,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(3.6)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
